test/test_ops.py

```diff
@@ -1577,6 +1577,121 @@ def is_bit_set(x):
             torch.is_complex,
         )
 
+    @onlyCUDA
+    def test_opinfo_aten_ops_coverage(self):
+        all_aten_ops = {
+            torch.ops.aten.__getattr__(attr)
+            for attr in dir(torch.ops.aten)
+            if not attr.startswith('__') and attr not in {'name'} and
+            isinstance(torch.ops.aten.__getattr__(attr), torch._ops.OpOverloadPacket)
+        }
+
+        # Copies inputs to inplace operations to avoid inplace modifications
+        #   to leaves requiring gradient
+        def _get_safe_inplace(inplace_variant):
+            @wraps(inplace_variant)
+            def _fn(t, *args, **kwargs):
+                return inplace_variant(t.clone(), *args, **kwargs)
+            return _fn
+
+        seen_aten_ops = set()
+
+        class CollectDispatchMode(torch.utils._python_dispatch.TorchDispatchMode):
+            def __torch_dispatch__(self, func, types, args=(), kwargs=None):
+                seen_aten_ops.add(func.overloadpacket)
+                kwargs = kwargs or {}
+                return func(*args, **kwargs)
+
+        for device in ["cpu", "cuda"]:
+            for op in op_db:
+                supported_dtypes = op.supported_backward_dtypes(device) or op.supported_dtypes(device)
+
+                if not supported_dtypes:
+                    continue
+
+                dtype = next(iter(supported_dtypes))
```

Comment:
Add a comment describing this dtype selection

Picking the first item is a little strange; this could use the same technique used to implement `ANY_ONE`:

https://github.com/pytorch/pytorch/blob/c407a7b20330afb957944ad26633a388220a4e43/torch/testing/_internal/common_device_type.py#L703